tools/deployment/package-helm/templates/redis-statefulset.yaml

```
@@ -69,39 +60,11 @@ spec:
             {{- include "clp.livenessProbeTimings" . | nindent 12 }}
             exec: *redis-health-check
       volumes:
-        - name: "config"
-          configMap:
-            name: {{ include "clp.fullname" . }}-config
-  persistentVolumeClaimRetentionPolicy:
-    whenDeleted: "Delete"
-  volumeClaimTemplates:
-    - metadata:
-        name: {{ include "clp.volumeName" (dict
+        - name: {{ include "clp.volumeName" (dict
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)